src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
```

Comment:
Conceptually it seems like the new "address validation" UI should be part of, or an alternate page in, the existing AddressEditModal. So rather than throwing up a new Modal after the user has submitted their address data, the AddressEditModal would render a different view. But I also see how that will require changing how the Vet360EditModal works since it expects a simple `render` prop and has no concept of changing what it renders depending on its props. Hmmm... Need to give this more thought. But what do you think of the idea in general?

Reply:
@erikphansen I don't think this will really line up well with the EditModal component and the EditModal component would have to be altered.  This could have larger implications and I don't think we should modify that component too much.

Reply:
I still would love to have the validation logic worked into the Vet360EditModals that contain the home and mailing address forms, but totally get why that's a substantial change that could have unintended side effects. But if we could do that we would not have to place the modal so high up on the component chain, which also _feels_ wrong to me.

Reply:
Side point - When we first implemented Vet360, we shared an an Address-edit modal w/the Letters application (Letters came first). However, addresses were in similar but different formats, so the Vet360 address was formatted before being passed to the address-edit modal, then formatted back to its original state after submit. At this point, Letters now uses Vet360, so there isn't really a need for this formatting at all, so there is an opportunity there to flatten out the logic by updating the address-edit to use the Vet360 address format.

(This was a long time ago though, so I could be remembering incorrectly)

But the gist is that the address-edit modal could probably use a solid update anyway.